style.css

```diff
@@ -48,23 +48,27 @@ footer a {
 }
 
 .vector {
-  margin-top: 17px;
+  margin-top: 10px;
   margin-right: 35px;
-  width: 18px;
-  height: 12px;
   right: 10%;
-  z-index: 5;
+  z-index: 1;
+  cursor: pointer;
+  font-size: 30px;
 }
 
-.vector span {
-  display: block;
+/* .vector.active {
+position:fixed;
+} */
+
+/* .vector span {
+  display: none;
   width: 100%;
   border: 1px solid #28352f;
-}
+} */
 
-.vector span:nth-child(2) {
+/* .vector span:nth-child(2) {
   margin: 2px 0;
-}
+} */

```

Comment:
- [ ]  You should not commit inline comments. They will make your project look messy.  remove all commented codes in your project to have clear and clean source codes.